filter/src/main/resources/reference.conf

```diff
@@ -0,0 +1,89 @@
+// Reference configuration of metric filters and predefined default values.
+//
+// - Metrics are organized to a tree structure that may split on any level.
+// - Root node is virtual and defines defaults that may be later overwritten on any level.
+// - Branch nodes may be used to configure subtree of metrics, they may also hold metrics.
+// - Leaf nodes are metrics like counters and timers that carry values.
+// - Each metric may compute and provide fields like count, rate, mean, percentiles, etc.
+// - Configurations from all levels are merged together, the most specific one wins.
+// - Separator for node names in this configuration is '/', separator used in real is implementation specific.
+// - The underlying format and parsing library is https://github.com/typesafehub/config.
+//
+// root (virtual)
+// |
+// +--- name1 --- + --- name2 --- ... --- nameN --- + --- myTimer
+// |              |                                 + --- myMeter
+// |              + --- haf --- ...
+// +--- ...
+//
+metricFiltersDefaults {
+  // Virtual root filter that is always defined.
+  root {
+    // Enable/disable flag for metrics reporting.
+    //
+    // (Future versions may extend the set of possible values to enable, error, warn, info, debug, trace, disable.
+    // Incompatible API changes would be needed.)
+    level = enable
+
+    // Counter fields to be reported.
+    counter {
+      count = enable
+    }
```

Comment:
Do we want to have these sections also for counters and gauges? They have only one field so they may be equally controlled by `level`. On the other hand the current variant is consistent with the rest of types.